keyboards/novem/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 Jose I. Martinez
```

Comment:
This file can be deleted.

Reply:
The last commit does not include this file, so we should be good now. 
tnx!